src/Codeception/Module/Yii2.php

```diff
@@ -460,8 +460,8 @@ public function _after(TestInterface $test): void
      */
     public function _failed(TestInterface $test, $fail): void
     {
-        $log = $this->yiiLogger->getAndClearLog();
-        if ($log !== '') {
+        $log = $this->yiiLogger?->getAndClearLog();
```

Comment:
Is this supported by all PHP versions currently supported by Yii2?

Reply:
CI will tell us xd